### PR TITLE
Cepheus-Class Mech Carrier! (CMM)

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
This ship adds the Cepheus-Class Mech Carrier! 

____

<details>

<summary>Image Dump</summary>

image

image

image

image

image

image

</details>

_____

it was first put into service a few years after the initial Xenomorph hives started cropping up om CMM worlds. While originally being made in response to Mech-equipped pirates, and exploration, it quickly got repurposed to fight the Xeno menace With its Sealed mechs. 

In recent years the need for the ship has declined with the production of specialized ships and better-equipped marines. Due to this, most of the vessels of its kind have fallen into disrepair, and are instead serving as stands in museums, or low-risk patrolling.

The in-game ship, while still in active duty, has fallen into slight disrepair, with Funds from central coming less and less frequently. This has led to the ship becoming mostly self-sufficient, relying on trading with other ships, and exploring/salvaging. 

the main equipment of the ship, would be the mechbay, and the machinery inside. They come equipped with a circuit imprinter, mech fabricator and a Security lathe, for printing weapons, and ammo for the Mechs. The armoury is also quite well stocked.
it does however in turn, lack quite a bit in medical equipment, and mining equipment (outside of mechs)

Crew:
1x Captain
2x Minuteman
2x Mech pilot
1x Roboticist
1x Engineer
3x cadet

Weapons: 3 m1911's / 1 cycler shotgun / captains Revolver (heavily limited on ammo)

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
 While cmm's ship rooster has been increasing, the removal of the Carina left a vacant spot for a carrier-esque ship. This ship fills that niche!

## Changelog

:cl:
add: Cepheus class
add: Minutemen Mech pilot job
add: Minutemen Roboticist job
/:cl:

### CODE REVIEW

**Comment by Valorium-RB** [comment]
I signed off on this, lore-wise.

File: _maps/shuttles/shiptest/minutemen_cepheus.dmm [hunk 1]
```diff
@@ -0,0 +1,5640 @@
...
+	pixel_x = 6;
+	pixel_y = 4
+	},
+/obj/item/pen{
+	pixel_y = 4;
+	pixel_x = 6
+	},
+/obj/machinery/light,
+/obj/item/phone{
+	desc = "Supposedly a direct line to Nanotrasen Central Command. It's not even plugged in.";
```
Comment (Erikafox) [inline _maps/shuttles/shiptest/minutemen_cepheus.dmm]: Minutemen ship, Nanotrasen?

**Review by Erikafox — COMMENTED** [review]
map review later

**Comment by Apogee-dev** [comment]
Now that some more CMM content has been merged, could you go ahead and replace relevant items with their new CMM equivalents? examples would be the cycler shotgun -> CM-15, ripley -> CMM ripley, and so on.

**Review by Erikafox — COMMENTED** [review]
looks good for tm

**Review by Erikafox — CHANGES_REQUESTED** [review]
Some doors should get names

The ship needs to not fly sideways

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Erikafox** [comment]
Did you fix it flying sideways

**Comment by Erikafox** [comment]
once it doesn't fly sideways and has a mech disk it's good to go.

**Review by Erikafox — APPROVED** [review]

**Comment by Erikafox** [comment]
I vaguely recall seeing it fly the right direction yesterday.

**Pull request merged**